### PR TITLE
Add a first post for topoguide comments.

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Relate https://github.com/c2corg/v6_forum/issues/44

### CODE REVIEW

File: script/import_scripts/punbb.rb [hunk 1]
```diff
@@ -361,10 +361,49 @@ def import_posts
...
+            document_id = tokens[0]
+            culture = tokens[1]
+            sql = "
+              SELECT name, module
+              FROM app_documents_i18n_archives
+              LEFT JOIN app_documents_archives
+                ON app_documents_archives.id = app_documents_i18n_archives.id
+              WHERE app_documents_i18n_archives.id = #{document_id};"
+            app_document = sql_query(sql).first
+            href = "https://www.camptocamp.org/#{app_document["module"]}/#{document_id}/#{culture}"
```
Comment (asaunier) [inline script/import_scripts/punbb.rb]: What about using a relative URL instead?

**Comment by asaunier** [comment]
Just to make sure: is a thread with an initial comment created for all guidebook documents (even the ones with no comment yet)?

File: script/import_scripts/punbb.rb (hunk 1 above)
Comment (arnaud-morvan) [inline script/import_scripts/punbb.rb]: Are you sure that forum url will be https://www.camptocamp.org/forum

**Comment by arnaud-morvan** [comment]
Yes, it create a first post for guidebook related topics with link to the document.

Note that the url I put in content is the V5 one, no document type mapping with V6 document types which seems to be a reduced set. Suppose that there will be permanent redirection from V5 documents to V6 ones.

**Comment by arnaud-morvan** [comment]
I create the first post only for existing guidebook related threads in punbb.

**Comment by asaunier** [comment]
> Note that the url I put in content is the V5 one, no document type mapping with V6 document types which seems to be a reduced set. Suppose that there will be permanent redirection from V5 document urls to V6 ones.

Can't we directly use the v6 URL instead of the old v5 ones?
There will probably be redirections from the old URLs, but until when? Maybe not forever.

Please note that several v5 URLs remain valid in v6: outings, routes, images, articles, books, xreports, maybe users?
The URLs that change: summits, sites, huts, access, products => become "waypoints".
So if you know the v5 modules you could replace them by the v6 ones for waypoints documents.

File: script/import_scripts/punbb.rb (hunk 1 above)
Comment (asaunier) [inline script/import_scripts/punbb.rb]: Most likely rather https://forum.camptocamp.org

**Comment by arnaud-morvan** [comment]
Are you sure this is comprehensive list of document url changes ?

**Comment by asaunier** [comment]
I think so.
Except that "access" are actually "parkings" in v5.
See the migration script here: https://github.com/c2corg/v6_api/tree/master/c2corg_api/scripts/migration/documents/waypoints

In v5:
http://www.camptocamp.org/summits
http://www.camptocamp.org/sites
http://www.camptocamp.org/parkings
http://www.camptocamp.org/huts
http://www.camptocamp.org/products
become waypoints.

**Comment by arnaud-morvan** [comment]
Yes, already search in migration script, but could miss something.
Are you telling that there will be an "access" view in V6 corresponding to V5 "parkings", not yet added.

**Comment by asaunier** [comment]
> Are you telling that there will be an "access" view in V6 corresponding to V5 "parkings", not yet added.

No. I used "access" by mistake. It's "parkings" in v5 and "waypoints" in v6 along with the other v5 point documents (summits, sites, huts, products).

Thanks for the change you have just made about the v5>v6 URLs.

Looks good to me.

**Comment by arnaud-morvan** [comment]
Thanks for review

**Pull request merged**